### ISSUE TITLE
Can not play video as fullscreen mode

### ISSUE THREAD

**Issue by atistank** [jvanakker/tvOSBrowser#62]
I face issue while play video that webview can not enter fullscreen mode. The website is not response anything when I touching

Is that because webview disable this mode ?

**Comment by jvanakker** [comment]
Yes UIWebView is officially not supported on tvOS by Apple, it's private. This app works around that, therefore it still has it's issues.

Spend some more time to get full screen video playback working. Added a private-runtime shim around the fullscreen video setup so the app no longer crashes and the fullscreen player UI now actually appears. Still didn't show the video. Then tried wiring in the player/view/layer state needed for rendering. But video still doesn't load, which suggests the remaining issue is deeper inside the private WebKit/AVKit fullscreen render path.

That could be a pretty big rabbit hole, so I'm abandoning the full screen video issue for now.
